pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -238,6 +239,23 @@ func (pl *TestPlugin) Bind(ctx context.Context, state *framework.CycleState, p *
 	return framework.NewStatus(framework.Code(pl.inj.BindStatus), injectReason)
 }
 
+func newTestCloseErrorPlugin(_ context.Context, injArgs runtime.Object, f framework.Handle) (framework.Plugin, error) {
+	return &TestCloseErrorPlugin{name: testCloseErrorPlugin}, nil
+}
+
+// TestCloseErrorPlugin implements for Close test.
+type TestCloseErrorPlugin struct {
+	name string
+}
+
+func (pl *TestCloseErrorPlugin) Name() string {
+	return pl.name
+}
+
+func (pl *TestCloseErrorPlugin) Close() error {
+	return errors.New("close err")
+}
```

Comment:
```suggestion
var closeErr = errors.New("close err")

func (pl *TestCloseErrorPlugin) Close() error {
	return closeErr
}
```

Reply:
@alculquicondor 
Thanks, but it doesn't solve this pointer issue because errs and fmt.wrapError have different pointers.
Without using .Error() method, do you have any good solutions?
(I guess using .Error() and comparing as string is easier way)


```
        -       &⟪0xc0003fc378⟫{
        -               errs: []error(⟪ptr:0xc000258f40, len:1, cap:1⟫{
        -                       &⟪0xc000192340⟫"fmt".wrapError{
        -                               msg: "test-close-error-plugin failed to close: close err",
        -                               err: &⟪0x0332abd0⟫"errors".errorString{s: "close err"},
        -                       },
        -               }),
        -       },
        +       &⟪0xc0004718a8⟫{
        +               errs: []error(⟪ptr:0xc000258880, len:1, cap:1⟫{
        +                       &⟪0xc000192060⟫"fmt".wrapError{
        +                               msg: "test-close-error-plugin failed to close: close err",
        +                               err: &⟪0x0332abd0⟫"errors".errorString{s: "close err"},
        +                       },
        +               }),
        +       },
```

Reply:
Don't wrap it. Just use `wantErr: closeErr`.

Reply:
I didn't know that, it works. Thanks!